### PR TITLE
DOC: Updated image and docs for SSO-Login

### PR DESCRIPTION
**Pull request by Abhinav-26**
# Description

Updated the image used in SSO Login Service, and changed few minor statements in docs.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# Checklist:

* [X] The title of the PR states what changed and the related issues number (used for the release note).
* [X] Does this PR require documentation updates?
* [X] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have tested it for all user roles

### CODE REVIEW

File: docs/user-guide/global-configurations/sso-login.md [hunk 1]
```diff
@@ -26,17 +26,15 @@ For examples see
...
 
-Only single SSO login configuration can be active at one time. Whenever you create or update any sso config that config will be activated and used by the system and previous configuration will be deleted.
-
-![](../../.gitbook/assets/sso-login-3.png)
+Only single SSO login configuration can be active at one time. Whenever you create or update any SSO config, it will be activated and used by the system and previous configurations will be deleted.
 
 URL and redirectURI should be same as provided in screenshots except domain substring.
 
-![](../../.gitbook/assets/sso-login-4.png)
+![](../../.gitbook/assets/sso-config)
```
Comment (prakarsh-dt) [inline docs/user-guide/global-configurations/sso-login.md:37]: Why does image file has no extension?

Comment (Abhinav-26) [reply docs/user-guide/global-configurations/sso-login.md:37]: Not required in this case, it will work.

Comment (prakarsh-dt) [reply docs/user-guide/global-configurations/sso-login.md:37]: Please add the jpg format as it helps in SEO purpose

Comment (Abhinav-26) [reply docs/user-guide/global-configurations/sso-login.md:37]: Added, the jpg extension.

Comment (prakarsh-dt) [reply docs/user-guide/global-configurations/sso-login.md:37]: You missed to update the link here

Comment (Abhinav-26) [reply docs/user-guide/global-configurations/sso-login.md]: updated.

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**